src/classes/CW_AssociateAudienceRewardHelperTest.cls

```diff
@@ -0,0 +1,124 @@
+@isTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.